Assets/Scripts/ConvergeGame/MultiConvergeGame.cs

```diff
@@ -144,35 +157,48 @@ public class MultiConvergeGame : MonoBehaviour
 	private int speciesRowCount;       // number of rows of species in ecosystem
 	private int speciesColFit;         // number of columns of species that will fit on screen
 	private int speciesColIndex;       // index of coloum of species presently starting display. Ranges 0 to speciesCount - speciesFit
+	private AudioSource audio;
+	private int playerWinner;
+	private string playerWinnerName;
+	private List<int> winners;         // lists by round the player id of the winner
+	private IDictionary roundsWon;     // Dictionary by player of number of rounds won
+	private int[] playerId; 
+	private int[] playerWinnings;
+	private int[] playerLastImprove;
 
 	void Awake ()
 	{
         tStamp = DateTime.UtcNow;
         sendNonHost = true;
         DontDestroyOnLoad (gameObject.GetComponent ("Login"));
         player_id = GameState.player.GetID ();
+		player_name = GameState.player.name;
 		balance = GameState.player.credits;
 		balBeg = balance;
-		Debug.Log ("Player name: " + GameState.player.name + ", credits: " + balance);
+		Debug.Log ("Player name: " + player_name + ", credits: " + balance);
 
         left = (Screen.width - width) / 2;
         top = (Screen.height - height) / 2;
         // DH change
         leftConfig = (Screen.width - widthConfig) / 2;
         topConfig = (Screen.height - heightConfig) / 2;
+		leftWinner = (Screen.width - widthWinner) / 2;
+		topWinner = (Screen.height - heightWinner) / 2;
         ftr1 = "";
         ftr1P = ftr1.Length * pixelPerChar;
 
         // DH change
-        OppViewWidth = (int)(width * 0.20f - 10);
-        Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);
-        buttonWidth = OppViewWidth > 125 ? 125 : OppViewWidth;
+		OppViewWidth = Math.Min((int)(width * 0.25f - 10), 160);
+		buttonWidth = OppViewWidth;
+        // Debug.Log ("Width / OppViewWidth: " + width + " " + OppViewWidth);
```

Comment:
We can remove this instead of commenting it out. Since we are using git for tracking history.